app/controllers/management/site_pages_controller.rb

```diff
@@ -125,6 +126,11 @@ def toggle_enable
   # Use callbacks to share common setup or constraints between actions.
   def set_site
     @site = Site.find_by({slug: params[:site_slug]})
+    routes = Route.where("site_id = ?", @site.id)
```

Comment:
This works, but the syntax used is usually this one: 

`Route.where site_id: @site.id`

In this situation, since the site has many roots (as per the "site.rb" file) you can just do (site.routes.count > 0)